drivers/Makefile.dep

```diff
@@ -47,14 +47,27 @@ ifneq (,$(filter bmp180,$(USEMODULE)))
   USEMODULE += xtimer
 endif
 
-ifneq (,$(filter bmx055,$(USEMODULE)))
-  FEATURES_REQUIRED += periph_i2c
+ifneq (,$(filter bme280_spi,$(USEMODULE)))
```

Comment:
Why not use the wildcard `%` and merge this one with the block under ? I think the wildcard could also be used in the block below that handles the distinction between spi and i2c.